doc/logos/README.md

```diff
@@ -0,0 +1,47 @@
+# scikit-learn Branding
+
+This section contains scikit-learn's branding standards.
+File types:
+- `SVG` Scalable Vector Graphics (SVG) are an XML-based markup language for describing two-dimensional based vector graphics. They can be created and edited with any text editor or with drawing software.
```

Comment:
For consistency, may the order of these items match the order the logos are shown below?

Reply:
Done.